src/display/canvas.js

```diff
@@ -1350,11 +1350,7 @@ const CanvasGraphics = (function CanvasGraphicsClosure() {
       // stroking alpha.
       ctx.globalAlpha = this.current.strokeAlpha;
       if (this.contentVisible) {
-        if (
-          strokeColor &&
-          strokeColor.hasOwnProperty("type") &&
-          strokeColor.type === "Pattern"
-        ) {
+        if (typeof strokeColor === "object" && strokeColor?.getPattern) {
```

Comment:
The optional chaining used here replaces an explicit `strokeColor !== null` check.

In practice there shouldn't ever be a situation when the `strokeColor` is `null`, as far as I can tell, but nonetheless I don't think the above will hurt.